ciao-controller/tables/workload_template.csv

```diff
@@ -4,3 +4,4 @@ eba04826-62a5-48bd-876f-9119667b1487,CNCI,test.yaml,efi,qemu,4e16e743-265a-4bf2-
 ca957444-fa46-11e5-94f9-38607786d9ec,Docker Ubuntu latest,docker-ubuntu.yaml,"",docker,fa7d86d8-fa46-11e5-8493-38607786d9ec,"ubuntu:latest",0
 ab68111c-03a6-11e6-87de-001320fb6e31,Docker Iperf,docker-iperf.yaml,"",docker,b5e696b8-03a6-11e6-a424-001320fb6e31,"mcastelino/iperf:latest",0
 79034317-3beb-447e-987d-4e310a8cf410,Boot Fedora23 from created volume based on image,test.yaml,legacy,qemu,"","", 0
+1cd4c4d9-ff84-4bfb-a371-a220b129977a,Boot Fedora23 from existing volume,test.yaml,legacy,qemu,"","", 0
```

Comment:
Why is this one in this patch but the "79034317-3beb-447e-987d-4e310a8cf410" in the previous patch?